cluster/minikube/mini_deploy.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash 
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2017 Red Hat, Inc.
+
+# get the config used widely in the kubevirt 
+docker_cmd_images="cmd/virt-controller cmd/virt-launcher cmd/virt-handler cmd/virt-api cmd/virt-manifest cmd/virt-dhcp cmd/virt-migrator" 
```

Comment:
converge with the configs in the hack folder. Make a config-minikube.sh if you need to.